packages/builder/src/stores/portal/licensing.js

```diff
@@ -130,11 +130,11 @@ export const createLicensingStore = () => {
       })
       actions.setUsageMetrics()
     },
-    willReachUserLimit: userCount => {
-      return willReachUserLimit(userCount, get(store).userLimit)
+    usersLimitReached: userCount => {
+      return usersLimitReached(userCount, get(store).userLimit)
     },
-    willExceedUserLimit(userCount) {
-      return willExceedUserLimit(userCount, get(store).userLimit)
+    usersLimitExceeded(userCount) {
+      return usersLimitExceeded(userCount, get(store).userLimit)
```

Comment:
Correct me if I'm wrong, but I've assumed that the prefix 'will...' is a reference to a future event, but now if limit is exceeded, is exceeded now.

Reply:
The `will` was added here as the users might not actually be exceeded. This is mostly used whenever we are in the process of adding new users via the portal, so that we can say 'the limit will be reached if you add 1 more user', so that we can show / hide the inputs for creating additional users. Same for CSV import.

Reply:
In your opinion, should I keep the 'will'? I mean, in the end the name of the function shouldn't reflect the external usage in use cases but the internal behaviour of the function. Just to confirm, what is the returning result of the function by itself? Forgetting the outside context, when that function returns `true` it means ('the limit is exceeded now' or 'the limit is not exceeded now but will be exceeded in the future')?